### ISSUE TITLE
Attack Chains doesn't seem to be enabled even for Bronze

### ISSUE THREAD

**Issue by nanaeda** [CodinGame/SpringChallenge2023#1]
I'm sorry if this is not the right place to report this. 
But, I feel the following part of the code should use LOSING_ANTS_CANT_CARRY not LOSING_ANTS_CANT_MOVE. The latter is always false while the former is false only during Wood, which is aligned with the rule description. 
I can also see the Attack Chains rule not being applied on game replays.

https://github.com/CodinGame/SpringChallenge2023/blob/5f73a72502b2994ed7469f0d6dff496caea3a2f0/src/main/java/com/codingame/game/Game.java#L172

**Comment by CGjupoulton** [comment]
You are correct. Additionally, attack chains weren't taken into account on the cell of the resource itself

**Comment by nanaeda** [comment]
Thank you for your check and quick fix!
